exasol_integration_test_docker_environment/lib/base/task_state.py

```diff
@@ -2,11 +2,11 @@
 
 
 class TaskState(Enum):
-    NONE = 0,
-    INIT = 1,
-    AFTER_INIT =2,
-    RUN = 3,
-    FINISHED = 4,
-    CLEANUP = 5,
-    CLEANED = 6,
+    NONE = 0
+    INIT = 1
+    AFTER_INIT = 2
+    RUN = 3
+    FINISHED = 4
+    CLEANUP = 5
+    CLEANED = 6
```

Comment:
fixed manually

Reply:
If you do not rely on specific numbers, the `auto` function within the `enum` package is a viable alternative to manual numbering.

Reply:
auto should be probably fine, I don't think, we have long time serialization of this enum